lib/office365/rest/event.rb

```diff
@@ -12,6 +12,12 @@ module Event
       def events(args = {})
         wrap_results(args.merge(kclass: Models::Event, base_uri: "/me/events"))
       end
+
+      def event(identifier)
+        raise ArgumentError, "Identifier must be provided" if identifier.nil? || identifier.empty?
+
+        wrap_results(kclass: Models::Event, base_uri: "/me/events/#{identifier}", get_by_id: true)
```

Comment:
@tiaki75019 

let's add an variable for base URI

```
BASE_URI = "/me/events".free

....

[BASE_URI, identifier].join('/')
```

Shall we use identifier to decide the response, I think it's enough

```
wrap_results(kclass: Models::Event, base_uri: [BASE_URI, identifier].join('/'), identifier: identifier)
```

Reply:
you are right

```
wrap_results(kclass: Models::Event, base_uri: [BASE_URI, identifier].join('/'), identifier: identifier)
```

Reply:
done